core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -186,8 +186,8 @@ class CityInfo {
         }
         for(unique in cityConstructions.builtBuildingUniqueMap.getUniques("Provides [] []")) { // E.G "Provides [1] [Iron]"
             val resource = getRuleset().tileResources[unique.params[1]]
-            if (resource != null)
-                cityResources.add(resource, unique.params[0].toInt(), "Buildings")
+            val amount = unique.params[0].toInt() * resource?.let { civInfo.getResourceModifier(it) }!!
```

Comment:
This will only give you the modifier.
Also, it's more readable to stick to if(resource!=null){ cityResources.add(resource, unique.params[0].toInt() * civInfo.getResourceModifier(resource), "Buildings") }